beluga/include/beluga/sensor/ndt_sensor_model.hpp

```diff
@@ -161,7 +160,7 @@ using NDTModelParam3d = NDTModelParam<3>;
 template <typename SparseGridT>
 class NDTSensorModel {
  public:
-  static_assert(std::is_same_v<SparseValueGrid<typename SparseGridT::map_type>, SparseGridT>);
+  // static_assert(std::is_same_v<SparseValueGrid<typename SparseGridT::map_type>, SparseGridT>);
```

Comment:
@serraramiro1 why comment this out? Should NDT sensor model changes be split into their own PR?

Reply:
This was a leftover from my side, it needed a very small update. Done.